std/file.d

```diff
@@ -862,7 +862,7 @@ unittest
 version(StdDdoc) void getTimesWin(R)(R name,
                                   out SysTime fileCreationTime,
                                   out SysTime fileAccessTime,
-                                  out SysTime fileModificationTime)
+                                  out SysTime fileModificationTime) @safe
```

Comment:
What's common practice in these situations? I thought we just added the missing attributes.